masterlist.yaml

```
@@ -25503,3 +25503,14 @@ plugins:
         crc: 0xB86273B2
         util: '[SSEEdit v4.0.4](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 35
+
+  - name: 'AzurasShield_SE_Eng.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/12998'
```

Links should have trailing slashes if possible for convenience & consistency:

```suggestion
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/12998/'
```